packages/components/src/components/Flyout/Flyout.tsx

```diff
@@ -113,6 +122,16 @@ export const FlyoutContent = (
                 className={flyoutContentStyles({ ...props })}
                 data-flyout-spacing={padding}
                 data-test-id={dataTestId}
+                onFocus={(e) => {
```

Comment:
I think we don't use initials for variable names

Reply:
fixed